src/meshapi/validation.py

```diff
@@ -61,10 +62,14 @@ def __init__(self, street_address: str, city: str, state: str, zip: int):
                 "text": self.address,
                 "size": 1,
             }
-            nyc_planning_req = requests.get(f"https://geosearch.planninglabs.nyc/v2/search", params=query_params)
+            nyc_planning_req = requests.get(
+                "https://geosearch.planninglabs.nyc/v2/search",
+                params=query_params,
+                timeout=DEFAULT_EXTERNAL_API_TIMEOUT_SECONDS,
+            )
```

Comment:
I know that #227 only covers the altitude endpoint, but I'm wondering if there's a way we can be resilient to the failure of this endpoint as well?

Maybe this is a larger concern that should be covered in a separate issue, but it would be nice if we could fall back to [our local](https://github.com/nycmeshnet/meshdb/blob/main/.env.sample#L13) [pelias](https://github.com/pelias/parser) container in these cases

Reply:
If you're going to look into pelias parsing, take a look at the import script which uses pelias extensively. You can probably re-use a lot of the logic

Reply:
I agree, was going to split out "graceful failure if geosearch.planninglabs.nyc is unavailable", but I can do that here. Also, I split out #304 to fall back to pelias.